apps/src/feedback.js

```diff
@@ -529,18 +521,10 @@ FeedbackUtils.prototype.displayFeedback = function (
   }
 };
 
-function logDialogActions(event, options, isPerfect) {
-  if (experiments.isEnabled(experiments.FINISH_DIALOG_METRICS)) {
-    firehoseClient.putRecord({
-      study: FIREHOSE_STUDY,
-      study_group: dialog_type,
-      event: event,
-      data_json: JSON.stringify({
-        level_type: options.level ? options.level.skin : null,
-        level_id: options.response ? options.response.level_id : null,
-        level_path: options.response ? options.response.level_path : null,
-        isPerfectBlockCount: isPerfect,
-      }),
+function recordShare(type, appType) {
+  if (Object.prototype.hasOwnProperty.call(EVENTS, type)) {
+    analyticsReporter.sendEvent(type, {
```

Comment:
I think the first argument here needs to be EVENTS[type], right? Or, maybe easier, just use `EVENTS.FINISH_SHARING_PUBLISH`, etc when you call `recordShare` (and then you don't need the if check above)?

Reply:
Oh yeah! That is more simple. Will just remove the condition and use the `EVENTS` values for the arguments.